### DISCUSSION TITLE
Where can I find a roadmap?

### DISCUSSION THREAD

**Discussion by jtuchel** [rome/tools#4753] [Q&A]
Hi there,
is there something like an official roadmap where I can find information about planned future releases?

Something like "in 2024, Rome provides the same tooling like Deno" ( just an example )

Thanks!

**Comment by Conaclos** [comment]
There is currently no official roadmap.
However, you can expect the support of CSS linting and formatting.

**Comment by jtuchel** [comment]
Hm okay :( 
It would be awesome to see support for the Compiler, Bundler and Testrunner soon, so I can use it for all my Node projects :)

**Reply by Conaclos** [reply]
Some works started around transformations. This is the foundation of the compiler.
